### PR TITLE
Prevent events on svg root

### LINKED ISSUE #1: This doesn't check against onload or similar attributes
Hi!
Thanks for this library, checking against scripts is nice, but there is also inline events.

Example:
```svg
<svg xmlns="http://www.w3.org/2000/svg" height="256" width="256" onload="alert('xss')">
	<rect ry=".5" rx=".5" height="256" width="256" fill="#0082c9"/>
	<g transform="translate(0 64)" stroke-width="22" fill="none" stroke="#fff">
		<circle r="26" cy="64" cx="40"/>
		<circle r="26" cy="64" cx="216"/>
		<circle r="46" cy="64" cx="128"/>
	</g>
</svg>
```

### PR DESCRIPTION
**Pull request by skjnldsv**
fix #1

I made the script compatible with non es6 browsers. Should works on ie11 for example :)
cc @mattkrick

### CODE REVIEW

**Comment by mattkrick** [comment]
Thanks for the PR!
Thankfully IE11 EOL is < 3 months away! 
For that reason, I can't accept the PR (a small package is more valuable than support of near-deprecated browsers)
but, feel to use this branch on your own. To include in your project, I love using https://github.com/ramasilveyra/gitpkg which easily lets you install a package right from a github tag.

**Comment by mattkrick** [comment]
Ahhhh I should've read the code entirely. This is still valid in evergreen browsers!
Great find. will publish a new version

**Comment by mattkrick** [comment]
live in 0.3.0, published on npm
(this PR used npm instead of yarn, so i made a separate PR to keep lockfiles consistent)

**Comment by skjnldsv** [comment]
> (this PR used npm instead of yarn, so i made a separate PR to keep lockfiles consistent)

OuUps, my bad!  :see_no_evil: 
Thanks for the update! :hugs: :wink:

File: src/sanitizeSVG.ts [hunk 1]
```diff
@@ -23,8 +24,12 @@ const sanitizeSVG = async (svg: File | Buffer, window = getWindow()) => {
   const div = window.document.createElement('div')
   div.innerHTML = svgText
   const svgEl = div.firstElementChild!
+  const attributes = [].slice.call(svgEl.attributes) || []
+  const events = attributes.filter(function (attr: Attr) {
+    return attr.name.indexOf('on') === 0
+  })
```
Comment (skjnldsv) [inline src/sanitizeSVG.ts:30]: Hey @mattkrick you introduced an issue.

https://github.com/mattkrick/sanitize-svg/blob/7ac6f9f8ccf5220f242b79daf02cb2fa7f8832a3/src/sanitizeSVG.ts#L27

Attributes are not strings, but `Attr`, meaning you get their value as a String with `attr.name`

Master is broken as well as 0.3.0

Comment (skjnldsv) [reply src/sanitizeSVG.ts:30]: https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.attr.html#name

Comment (skjnldsv) [reply src/sanitizeSVG.ts:30]: Capture d’écran_2020-09-08_21-21-36

Comment (mattkrick) [reply src/sanitizeSVG.ts:30]: oops! good catch, should've tested...

Comment (mattkrick) [reply src/sanitizeSVG.ts:30]: k, all better

**Pull request closed without merging**